packages/FlexGrid/Col/Col.jsx

```diff
@@ -5,52 +5,111 @@ import { Subscriber } from 'react-broadcast'
 import { Col as ReactFlexboxGridCol } from 'react-flexbox-grid'
 
 import safeRest from '../../../shared/utils/safeRest'
+import { deprecate } from '../../../shared/utils/warn'
 import styles from './Col.modules.scss'
 
-const removeProps = ({
-  xs,
-  sm,
-  md,
-  lg,
-  xl,
-  xsOffset,
-  smOffset,
-  mdOffset,
-  lgOffset,
-  xlOffset,
-  ...rest
-}) => safeRest(rest)
-
-const Col = ({ span, offset, children, ...rest }) => (
-  <Subscriber channel="flex-grid">
-    {gutter => (
-      <ReactFlexboxGridCol
-        {...removeProps(rest)}
-        xs={span || true}
-        xsOffset={offset}
-        className={gutter ? styles.padding : styles.gutterless}
-      >
-        {children}
-      </ReactFlexboxGridCol>
-    )}
-  </Subscriber>
-)
+const Col = ({ children, span, offset, ...rest }) => {
+  if (offset) {
+    deprecate(
+      'core-flex-grid',
+      `The offset prop is deprecated due to the addition of the new responsive
+    offset props.
+    Replace offset in your Col definition with xsOffset for identical functionality.`
+    )
+  }
+  if (span) {
+    deprecate(
+      'core-flex-grid',
+      `The span prop is deprecated due to the addition of the new responsive props.
+      Replace span in your Col definition with xs for identical functionality.`
+    )
+  }
 
+  const props = rest
```

Comment:
This is still causing a modification of `rest` though, which is not ideal. Best to make a copy so that the incoming object is not modified accidentally.

```js
const props = {...rest}
```